blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubOrgFolderPermissionsTest.java

```diff
@@ -0,0 +1,222 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.jvnet.hudson.test.MockAuthorizationStrategy;
+import org.jvnet.hudson.test.MockFolder;
+import org.jvnet.hudson.test.TestExtension;
+
+import com.cloudbees.hudson.plugins.folder.Folder;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.mashape.unirest.http.exceptions.UnirestException;
+
+import hudson.model.Item;
+import hudson.model.ItemGroup;
+import hudson.model.TopLevelItem;
+import hudson.model.User;
+import io.jenkins.blueocean.rest.factory.organization.OrganizationFactory;
+import io.jenkins.blueocean.rest.model.BlueOrganization;
+import io.jenkins.blueocean.service.embedded.OrganizationFactoryImpl;
+import io.jenkins.blueocean.service.embedded.rest.OrganizationImpl;
+import jenkins.branch.OrganizationFolder;
+import jenkins.model.Jenkins;
+import jenkins.model.ModifiableTopLevelItemGroup;
+
+
+public class GithubOrgFolderPermissionsTest extends GithubMockBase {
+
+    @Test
+    public void canCreateWhenHavePermissionsOnDefaultOrg() throws Exception {
+        MockAuthorizationStrategy authz = new MockAuthorizationStrategy();
+        authz.grant(Jenkins.ADMINISTER).everywhere().to(user.getId());
+        j.jenkins.setAuthorizationStrategy(authz);
```

Comment:
the jwt token becomes invalid.  
a call to `jwtToken = getJwtToken(j.jenkins, "vivek", "vivek");` seems to fix the issue.
`